conf/metadata.php

```diff
@@ -5,6 +5,7 @@
  * @author Wolfgang Gassler <wolfgang@gassler.org>
  */
 
+$meta['initRepo'] = array('onoff');
 $meta['pushAfterCommit'] = array('onoff');
 $meta['periodicPull'] = array('onoff');
```

Comment:
This cannot work anymore in this new setup. We need diverging here.

Either it's one (or a given number of) git repository/ies and we can push to it (them) and periodically pull from it (them). Or we don't known how many we have, or how they are structured, and the pulling must be done from a cron job that understands the git repository structure. The commit + push is possible however.

I think it's saner : do one with the right tool. Thought ?

Reply:
@ochurlaud I did work on the merge of your changes and I did fix the open issues and updated the entire stuff a bit by changing your `initRepo` config into the new `autoDetermineRepos` config with default set to 1 and appropriate description. This inverts your logic, but it is more self explaining IMO by reflecting what it is supposed to do. This config should work also for backward compatibility out of the box for existing installations. I did test in my environment for multiple repos beyond different namespaces - and it works as exepcetd so far. All is merged currently on this repos branch [issues/47_multiple_repos](https://github.com/woolfg/dokuwiki-plugin-gitbacked/tree/issues/47_multiple_repos) for your review. Can you please check and test, if it works for you as you expected ?

Reply:
Your branch is cleaner. I'll test that when I find some time. It might take a week or so.

Reply:
@ochurlaud - DONE - see #74